tsconfig.json

```diff
@@ -12,7 +12,6 @@
     "noFallthroughCasesInSwitch": true,
     "noUnusedLocals": true,
     "noImplicitReturns": true,
-    // change to true when all files are switched over to ts
```

Comment:
Should we also change the value to true, per the comment being removed?

Reply:
Yes, made the change.  Compiling will be blocked by TS errors.